openedx/features/caliper_tracking/tests/expected/edx.course.enrollment.activate.json

```diff
@@ -0,0 +1,41 @@
+{
+  "@context": "http://purl.imsglobal.org/ctx/caliper/v1p1",
+  "id": "urn:uuid:2635b9dd-0061-4059-ac61-2718ab366f75",
+  "type": "Event",
+  "actor": {
+    "id": "edx.devstack.lms:18000/u/ali",
+    "type": "Person"
+  },
+  "action": "Activated",
+  "object": {
+    "id": "course-v1:edx+cs-101+2018",
+    "type": "CourseOffering"
+  },
+  "eventTime": "2018-11-01T03:35:04.214775",
+  "referer": "http://localhost:18000/courses/course-v1:edx+cs-101+2018/about",
+  "session": "8205ce7ea40a0619955e15f072dc33e2",
+  "extensions": {
+    "extra_fields": {
+      "username": "ali",
```

Comment:
`username` will be incorporated into `actor` object. There is no need to list it here.

Reply:
Moved to the object field in the latest commit

Reply:
Please look at the comment related to this.